src/stopwords.c

```diff
@@ -147,3 +147,32 @@ void StopWordList_RdbSave(RedisModuleIO *rdb, StopWordList *sl) {
   }
   TrieMapIterator_Free(it);
 }
+
+void ReplyWithStopWordList(RedisModuleCtx *ctx, struct StopWordList *sl) {
+  RedisModule_ReplyWithSimpleString(ctx, "stop_word_list");
+  /*
+  if (sl == __default_stopwords) {
```

Comment:
this either works or it doesn't. Why is this commented out?

Reply:
@mnunberg 
If the default is used, I don't print anything. It is checked where the function is called.
Remove altogether so it can potentially be used to print the default list.